### DISCUSSION TITLE
Error: <path> attribute d: Expected number, "MNaN NaNCNaN NaN …".

### DISCUSSION THREAD

**Discussion by mustela** [apexcharts/apexcharts.js#3613] [Q&A]
Hey all! 
Im trying to understand what the issue is with this chart.  https://codepen.io/erjankowski/pen/gOjvMVz

The chart is being displayed but the console is shown lot of errors... All the datetime values and values are correct, so not really sure what's going on.

![Screenshot 2023-01-22 at 16 43 04](https://user-images.githubusercontent.com/325222/213924873-0255afd1-9e8a-424b-bbbf-8bb54fbf66c0.png)

Any idea? 

Thanks!

**Comment by Rao-Mudasir** [comment]
Any solution you got??

**Reply by mustela** [reply]
I just update the library and it got solved. https://github.com/apexcharts/apexcharts.js/issues/3615

**Comment by mustela** [comment]
Fixed here https://github.com/apexcharts/apexcharts.js/issues/3615
